app/models/user.rb

```diff
@@ -77,6 +78,8 @@ class User < ActiveRecord::Base
   belongs_to :language
 
   belongs_to :org
+  
+  belongs_to :department
```

Comment:
I would add the optional flag here unless the plan is for every org to have a default department: `belongs_to :department, optional: true`

Reply:
@briri I had to use `belongs_to :department, required: false` as  `:optional` only in `Rails 5`.

Reply:
👍 sorry. I had been reading up on Rails 5 recently. Its hard to keep these things straight sometimes ;)

Reply:
@briri Good to know about when we upgrade.